lib/modules/hideChildComments.js

```diff
@@ -74,10 +79,12 @@ module.go = () => {
 
 function toggleComments(action, obj) {
 	let commentContainers;
-	if (obj) { // toggle a single comment tree
-		commentContainers = $(obj).closest('.thing');
-	} else { // toggle all comments
+	if (obj === true) { // toggle only top level comments
```

Comment:
Instead of using the second param for two things, I would break `toggleComments` into three functions, something like:
```js
function toggleComments(action, containers) {
    containers::forEachChunked(container => {
        ...
    });
}

function toggleIndividualComment(action, ele) {
     toggleComments(action, $(ele).closest('.thing'));
}

function toggleAllComments(action, includeChildren) {
     const containers = includeChildren ?
        document.querySelectorAll('div.commentarea div.sitetable div.thing') :
        document.querySelectorAll('div.commentarea > div.sitetable > div.thing');

    toggleComments(action, containers);
}
```